docs/source/images/diagram.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Over/under "Output 1" there is a stray "z" somehow

FYI this SVG only seems to work fine in browsers; it doesn't load for example in inkscape